guides/metrics.md

```diff
@@ -106,6 +106,20 @@ live_dashboard "/dashboard", metrics: MyAppWeb.Telemetry
 
 Now refresh the "/dashboard" page and the metrics functionality should be enabled. Each metric goes to a distinct group based on the metric name itself.
 
+# Providing history for metrics
+
+If you wish to populate metrics with current historical data saved from telemetry or another data source,
+modify the config above like so:
+
+```elixir
+live_dashboard "/dashboard",
+    metrics: MyAppWeb.Telemetry,
+    historical_data: {MyStorage, :historical_metric_data, []}
```

Comment:
```suggestion
  metrics: MyAppWeb.Telemetry,
  historical_data: {MyStorage, :historical_metric_data, []}
```

Reply:
This example uses `historical_metric_data`, the other guide says `historical_data`. We should probably keep this and the next paragraph in sync with the previous section.

Reply:
ugh, yes, I've been a little back and forth in both trying to make guides clear, and clear that there's no assumption about what the function name is here... also had a debate whether to stick with "historical_data" as the primary config key here or whether it would be better to call it just "history" or perhaps "metrics_history".  Let me know if you have any preference/opinion.  I'll go through and add all these suggestions to a batch and accept there for now of course and then try and work later today on the stuff below that needs changing but doesn't have "suggestions" in github.